packages/webapp/pages/squads/create.tsx

```diff
@@ -120,7 +120,10 @@ function CreatePost(): ReactElement {
   const { onCreateSquad, isLoading } = useSquadCreate({
     onSuccess: (newSquad) => {
       const form = formToJson<CreatePostProps>(formRef.current);
-
+      /* If the squad gets created, the squad list gets updated and then the selected index becomes the last squad in the list.
+      Therefore on success, we set it to the first item in the list, which is the newly created squad.
+      */
+      setSelected(0);
```

Comment:
This is mostly just to make the UI not look janky, as the time between the squad being created and navigation happening might take a second, which causes this to happen:


https://github.com/user-attachments/assets/f1df7f38-1436-498d-b44b-0792088086a8